src/cmd/dut.rs

```diff
@@ -260,6 +280,7 @@ lazy_static! {
         let mut m: HashMap<&'static str, DutAction> = HashMap::new();
         m.insert("wait_online", Box::new(do_wait_online));
         m.insert("reboot", Box::new(do_reboot));
+        m.insert("reboot_other", Box::new(do_reboot_other));
```

Comment:
so replace this with `toggle_boot_partition`

Reply:
I changed it to "switch_to_secondary", since what the command does is not actually toggling the partition. If you run the command twice, the higher priority is still given to the secondary partition.

Reply:
I see. In that case, how about adding a switch for the opposite direction (boot from primary) when the user wants to get back to the original one? (I'm not sure how you can get back to the normal boot order without adding such command (except for manual re-flashing...))

Reply:
Okay, I can add another sub-command for that. I think it is better than actually implementing the toggle command, which is doable but maybe more confusing since that forces you to know the current priority order.  In contrast, `switch_to_primary/secondary` sub-commands can be issued without knowing that.
If you really want to go for the `toggle`, please let me know.

---
Well, in theory, you can reboot the DUT and `switch_to_secondary` and reboot again to get back to the original set... (Yeah, of course, I do not mean it)

Reply:
I just uploaded a version with `switch_to_primary` and `switch_to_secondary`.  Does this look better?

Reply:
yes, it looks good to me now. thanks for the fix!